baybe/utils/basic.py

```diff
@@ -144,3 +146,25 @@ def __init__(self, fn: Callable) -> None:
 
     def __get__(self, _, cl: type):
         return self.fn(cl)
+
+
+def refers_to(cls: type, name_or_abbr: str, /) -> bool:
+    """Check if the given name or abbreviation refers to the specified class."""
+    return name_or_abbr in (
+        (cls.__name__, cls.abbreviation)
+        if hasattr(cls, "abbreviation")
+        else (cls.__name__,)
+    )
+
+
+def find_subclass(base: type, name_or_abbr: str, /):
+    """Retrieve a specific subclass of a base class via its name or abbreviation."""
+    from baybe.utils.basic import get_subclasses
```

Comment:
Why don't you import this at top level?

Reply:
thanks for the catch, was an artifact that was necessary before I moved the function to a different location